frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/FastForwardMerge.java

```diff
@@ -38,23 +41,24 @@ private static void mergeNonCurrentBranch(Project project,
     val successFFMergeNotification = fetchNotificationPrefix
         + getString("action.GitMachete.BaseFastForwardMergeBranchToParentAction.notification.title.ff-success")
             .format(stayingName, movingName);
-    @SuppressWarnings("regexp") val failFFMergeNotification = fetchNotificationPrefix.replaceAll("<br/>", "\n")
-        + getString("action.GitMachete.BaseFastForwardMergeBranchToParentAction.notification.title.ff-fail").format(stayingName,
+    String failFFMergeNotification = changeNewLineSign(fetchNotificationPrefix)
```

Comment:
Awww I see now.... actually, how about passing the `<br/>` suffix (or an empty string) in a separate param? could even be made a boolean param (`insertNewline` or sth)

Or even better, is this suffix **always** appended to `fetchNotificationPrefix`?

Reply:
nope because in one place fetchNotificationPrefix is "" and we do not want to add` <br/>`in that situation

Reply:
Okay, so I'd suggest a separate (boolean?) param then